pkg/connection/connectionset.go

```diff
@@ -1,5 +1,8 @@
-// Copyright 2020- IBM Inc. All Rights Reserved.
-// SPDX-License-Identifier: Apache-2.0
+/*
+Copyright 2023- IBM Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
```

Comment:
Only the start year needs to be changed (2023). There's no need for longer comment.

Reply:
Preferred to use the same header style as in the other repos, so it is more consistent.